src/native/libs/System.Security.Cryptography.Native/configure.cmake

```diff
@@ -9,8 +9,12 @@ check_function_exists(
     HAVE_OPENSSL_EC2M)
 
 check_function_exists(
-	SSL_get0_alpn_selected
-	HAVE_OPENSSL_ALPN)
+    SSL_get0_alpn_selected
+    HAVE_OPENSSL_ALPN)
+
+check_function_exists(
```

Comment:
I don't think we want this. With portable SSL we need to detect this dynamically during dlopen().

Reply:
I tried to follow existing examples, can you recommend better way of handling older OpenSSL versions? The functions I need are from 1.1.1 onward.